### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by melodysdreamj**
### Change Safety

- [ ] I have added tests to cover my changes
- [ ] I have considered potential risks and mitigations for my changes

### CODE REVIEW

**Review by richbeales — CHANGES_REQUESTED** [review]
Please resubmit without your .idea/ and inspectionProfiles/ directories.

File: .idea/.gitignore
Comment (nponeccop) [inline .idea/.gitignore:1]: What are all these files? Remove them and keep the PR scope low

**Review by nponeccop — CHANGES_REQUESTED** [review]
You need to remove all the extra files you added. If they are beneficial, submit them as separate PRs.

**Comment by nponeccop** [comment]
@melodysdreamj Implement the suggestions and resolve the conflicts

**Pull request closed without merging**